docs/sources/mimir/send/otel-exponential-histograms/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Learn how to collect and send exponential histograms with the OpenTelemetry Collector
+menuTitle: OpenTelemetry exponential histograms
+title: Collect and send exponential histograms with the OpenTelemetry Collector
+weight: 100
+---
+
+# Collect and send exponential histograms with the OpenTelemetry Collector
+
+OpenTelemetry [exponential histograms](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exponentialhistogram) are compatible with Prometheus native histograms. The only significant difference is that exponential histograms store the `min` and `max`  observation values explicitly, but native histograms don't, which means you need to estimate the minimum and maximum by using the 0.0 and 1.0 quantile respectively. 
```

Comment:
Do you mean you need to estimate the min and max values for native histograms? If that's the case, can we rephrase this to indicate that for exponential histograms, you don't need to estimate these values, since they are explicitly stored?

Since this topic if about exponential histograms, I think it makes more sense to have that be the focus.